calico-cloud/image-assurance/understanding-scan-results.mdx

```diff
@@ -59,6 +59,18 @@ To change the CVSS threshold values, note the following:
 - Changes to threshold values take effect immediately and alter the scan results for images already in the system
 - If you are using admission controller policies, changing a value may allow pods in your Kubernetes cluster that were previously being blocked, to now be deployed or vice versa.
 
+## Exploit Data
+
+The scanning process also attaches EPSS and known exploit data onto each image and vulnerability viewable through the UI.
+
+An EPSS score of 0.1 - or 10% - means a vulnerability has a 10% probability that the vulnerability will be exploited in the wild within the next 30 days. 
```

Comment:
Minor: The word "vulnerability" is used twice in quick succession and does not read well. 

I suggest taking out the second occurrence of "vulnerability" and replace it with "it". 

So change it to something like: 

> An EPSS score of 0.1 - or 10% - means a vulnerability has a 10% probability **it** will be exploited in the wild within the next 30 days.